pkg/util/util.go

```diff
@@ -84,7 +84,7 @@ func GenerateRandomString(n int) string {
 	rand.Seed(time.Now().UnixNano())
 	b := make([]rune, n)
 	for i := range b {
-		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+		b[i] = letterRunes[rand.Intn(len(letterRunes))] // #nosec G404
```

Comment:
once https://github.com/openshift/odo/pull/3358 is in you probably will have to rebase

Reply:
#3358 has merged, and I've rebased from master

Reply:
Cool :+1: